cnames_active.js

```diff
@@ -568,6 +568,7 @@ var cnames_active = {
   "escape": "licshee.github.io/Escape.js", // noCF? (don´t add this in a new PR)
   "esfx": "esfx.github.io",
   "esper": "codecombat.github.io/esper.js",
+  "eth": "https://eth-js.github.io/eth-dev-tools/#/",
```

Comment:
```suggestion
  "eth": "eth-js.github.io/eth-dev-tools",
```
Please use the correct target format like everyone else.

Reply:
updated